generator/testdata/rust/gclient/golden/iam/v1/src/lib.rs

```diff
@@ -21,6 +21,8 @@ use gax::error::{Error, HttpError};
 use google_cloud_auth::{Credential, CredentialConfig};
 use std::sync::Arc;
 
+pub mod client;
```

Comment:
Is the client implementing the trait going to be in a separate PR?

Reply:
yes, but I can make it a single PR if you think that is easier to grok.

Reply:
See #356 if you are interested.